documentation/source/_static/css/custom.css

```diff
@@ -9,5 +17,286 @@
 .wy-table-responsive {overflow: visible !important;}
 .wy-table-responsive table td, .wy-table-responsive table th  {white-space: normal; !important;}
 
+.tutorials-header .header-logo {
+  background-image: url("../../../../imgs/logo_sct.png");
```

Comment:
should this also move to `source/_static/img`?

Reply:
Done: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3355/commits/b4666cd317a8496f8663c7c6f07e1d763e8c0129